src/Migration/Validation/SwagMigrationValidationService.php

```diff
@@ -70,12 +71,12 @@ public function validate(MigrationContextInterface $migrationContext, Context $s
             $this->validateFields($context);
             $this->validateAssociations($context);
         } catch (\Throwable $e) {
-            $this->loggingService->addLogEntry(
+            $context->getValidationResult()->addLog(
                 SwagMigrationLogBuilder::fromMigrationContext($context->getMigrationContext())
                     ->withEntityName($context->getEntityDefinition()->getEntityName())
                     ->withExceptionMessage($e->getMessage())
                     ->withExceptionTrace($e->getTrace())
```

Comment:
Also adding `convertedData` if it's available could be helpful to troubleshoot the exception:
```php
->withConvertedData($context->getConvertedData())
```